src/components/ui/Hero/hero.scss

```diff
@@ -59,6 +59,11 @@
   }
 
   [data-hero-image] {
+    // override 50% padding top from component
+    [aria-hidden][style]:first-child {
+      padding-top: 100% !important; /* stylelint-disable-line declaration-no-important */
```

Comment:
I haven't noticed any visual difference. So, I didn't get this padding, does it make any other significant change? 😕 
`padding: 50%`
![image](https://user-images.githubusercontent.com/3356699/156840007-693282bc-184b-4594-a55f-c67bfad4f7ba.png)

`padding: 100%`
![image](https://user-images.githubusercontent.com/3356699/156840092-1af1f551-95b7-465e-a2c1-3aa856d9f96e.png)